### PR TITLE
generically block almost all Lucky visitor scam and fake video scam

### PR DESCRIPTION
**Pull request by Yuki2718**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

https://github.com/AdguardTeam/AdguardFilters/issues/139667
https://github.com/AdguardTeam/AdguardFilters/issues/135963
etc.

#### If possible, a screenshot of a page or application should not be cropped too much. Otherwise, it is not always clear where the element is located

1. Your comment

I once suggested this but it's rejected as I recall. A contributor @SeriousHoax brought my attention to this in response to my [tweet](https://twitter.com/Yuki27183/status/1626935791778922497) explaining uBO blocks 99 % of Lucky visitor scam by the rule. I could easily find issues which could have been blocked as the example above, and can say Browsing Security rarely blocks them. The rule has been in uBO badware and DandelionSprout's Anti-Malware list for years now with occasional adjustment and there was only one false positive due to too rough regex in the very early version. I omitted URL as nowadays their life becomes shorter and shorter - the link in the tweet got blocked by GSB few hours later of the tweet but at the time was already dead. If you need a fresh URL to test, I'll PM so that you can check within hours.

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

**Comment by Alex-302** [comment]
Where can user come across sites like this?

**Comment by Yuki2718** [comment]
> Where can user come across sites like this?

The most common source is search engine such as Google, Bing, or DDG. No need to search for something special, you just need to be (un)lucky.

**Comment by Alex-302** [comment]
I don't like generic regexp rules, if the rule is to be used rarely.

**Comment by Yuki2718** [comment]
> if the rule is to be used rarely.

Who knows, Lucky visitor is probably one of the most common scam.

**Comment by Alex-302** [comment]
I often have to visit questionable sites or google something, but I find malicious sites mostly in email spam.

**Comment by Yuki2718** [comment]
I'm not a security expart so has no idea how popular Lucky visitor scam is. Apparently Lucky Visitor is a Japanese nickname, if your search language is English, search for `Annual Visitor Survey` or `Browser Opinion survey` and you'll see many articles.

**Comment by krystian3w** [comment]
This can be delivered by someting like https://martinhaunschmid.com/dollyrat-sophisticated-wordpress-malware on unprotected Wordpress pages - pined by event listener  to `<html>` or `<body>` so steal all clicks.

**Comment by Yuki2718** [comment]
Yep, you really don't need to visit questionable sites to see them. Just search common terms and click one of the results, this is how user gets them.

**Comment by Yuki2718** [comment]
Found List-KR already has a similar rule: `/^https:\/\/[A-z0-9\.]*[A-z0-9]+\.[A-z]+\/\?u=[A-z0-9]{7}&o=[A-z0-9]{7}/$document` @piquark6046 

Here when you searched for `adguard` in Japanese:

<details>
<summary>Screenshot</summary>

visitor

</details>

**Comment by piquark6046** [comment]
Did you mean the following issue?
 - https://github.com/List-KR/List-KR/issues/598
 - https://github.com/AdguardTeam/AdguardFilters/issues/141681
 - https://github.com/uBlockOrigin/uAssets/issues/16596

**Comment by Yuki2718** [comment]
Didn't know you opened issues. Yes, ongoing for years and I can trivially find these scam.

**Comment by krystian3w** [comment]
It depends on how advanced the ban can be within the AdGuard service: https://adguard.com/kb/general/browsing-security/

image

Eset can ban path.

I have similar limitation like AdGuard Home then wildcard and regex can't read params.

**Comment by Yuki2718** [comment]
IDK the granularity of Browsing Security, the only thing I can say for sure is it almost never blocks these in real-time. And I'm almost sure another victims is falling while we are disgussing this.

**Comment by Yuki2718** [comment]
> if the rule is to be used rarely.

Isn't the frequency of reports speaking?
https://github.com/AdguardTeam/AdguardFilters/issues/129112
https://github.com/AdguardTeam/AdguardFilters/issues/135114

User's voice:
> this is a very famous malicious website,why it is still not blocked by malware protection?

**Comment by krystian3w** [comment]
My baby live:

```
hXXps[:]//2614.babyebbwow[.]live/web/?sid=t3~hyeknjrrdtp30fpininya4j2
```

based on KAD report.

**Comment by krystian3w** [comment]
@Yuki2718 I guess we can again use `domain=` for regexes to strict blocking:

obraz

But area will be with enabled scrolling...:

obraz | obraz | obraz
---- | --- | ---
Part "1/3" | Part "2/3" | rest of Part

**Comment by krystian3w** [comment]
But with used `domain=TLD-s` can by updated by "copy-paste" instead "think-paste" and regex will be limited at diffrent level than "tokenization".

**Comment by Yuki2718** [comment]
I'll add domain= if requested.

**Comment by Yuki2718** [comment]
https://github.com/AdguardTeam/AdguardFilters/issues/144262

**Comment by Yuki2718** [comment]
> I'll add domain= if requested.

Or it may be good not to limit TLDs as done by @piquark6046 , I'm just not sure if that's safe. Also removing `(?!www)` may worth so that the rule works on platforms not supporting lookahead.

File: BaseFilter/sections/general_url.txt [hunk 1]
```diff
@@ -15,6 +15,8 @@
 ! Banner names
 !
 !
+! Lucky visitor and video scam ex. https://1821.antbenjaw.live/jiehxxgr/?u=1nup806&o=0wywy2l&t=...
+/^https?:\/\/(?!www)[0-9a-z]*\.?[-0-9a-z]{4,}\.[a-z]{2,11}\/(?:[0-9a-z]{6,8}\/)?\/?\?[ou]=[0-9a-z]{7}&[ou]=[0-9a-z]{7}/$document,match-case,domain=beauty|buzz|com|de|fun|guru|im|info|life|live|mobi|motorcycles|online|pw|quest|shop|site|skin|space|top|us|xyz|yachts
```
Comment (Alex-302) [inline BaseFilter/sections/general_url.txt:19]: `com|de` makes whole `$domain` redundant.

Comment (Yuki2718) [reply BaseFilter/sections/general_url.txt:19]: Okay, will remove the domain part.

Comment (Alex-302) [reply BaseFilter/sections/general_url.txt]: Should not `&t=` be matched?

**Review by Alex-302 — APPROVED** [review]

**Comment by Yuki2718** [comment]
Thanks!

**Pull request merged**